libpod/container_internal_linux.go

```diff
@@ -48,23 +50,55 @@ func (c *Container) unmountSHM(mount string) error {
 // prepare mounts the container and sets up other required resources like net
 // namespaces
 func (c *Container) prepare() (err error) {
+	var (
+		wg                              sync.WaitGroup
+		netNS                           ns.NetNS
+		networkStatus                   []*cnitypes.Result
+		createNetNSErr, mountStorageErr error
+		mountPoint                      string
+	)
+
+	wg.Add(2)
```

Comment:
Safer to do a .Add(1) at the beginning of each go func(), in case we need to add another to prepare

Reply:
done